multidim-arrays/bigarray_test.ml

```diff
@@ -3,17 +3,28 @@
  * In this example, tensors are represented as arrays of arrays of arrays.
  *)
```

Comment:
Please, move the changes to a separate file and update the comment above to describe your alternative approach. Note that the original comment (tensors are represented as arrays of arrays of arrays) was wrong. I fixed it in the new branch of the repository.